2016-09-09 14:46:41:776  INFO BATTLE:main - effect模块初始化开始。
2016-09-09 14:46:42:791  INFO BATTLE:main - effect模块初始化完成。
2016-09-09 14:46:42:834  INFO BATTLE:main - battle模块初始化开始
2016-09-09 14:46:42:912  INFO BATTLE:main - battle阵型信息初始化完成
2016-09-09 14:46:43:287  INFO BATTLE:main - battle AI信息初始化完成
2016-09-09 14:46:43:318  INFO BATTLE:main - 战场信息初始化完成
2016-09-09 14:46:43:318  INFO BATTLE:main - battle模块初始化完成
2016-09-09 14:46:45:982  INFO BATTLE:main - BUFF模块初始化开始
2016-09-09 14:46:46:435  INFO BATTLE:main - BUFF模块初始化完成
2016-09-09 14:47:20:248 ERROR BATTLE:mkdb.Worker.procedure.51 - 在角色4097下线的情况下，使用了该角色的ERole。因为ERole是内存表，数据可能没有  ，请根据堆栈内容确认此使用是否正确。
2016-09-09 14:47:20:261 ERROR BATTLE:mkdb.Worker.procedure.51 - 	at java.lang.Thread.getStackTrace(Unknown Source)
	at fire.pb.effect.RoleImpl.<init>(RoleImpl.java:80)
	at fire.pb.skill.SceneSkillRole.equip(SceneSkillRole.java:165)
	at fire.pb.role.PCreateRole.process(PCreateRole.java:503)
	at mkdb.Procedure.call(Procedure.java:109)
	at mkdb.Transaction.perform(Transaction.java:279)
	at mkdb.ProcedureFuture.run(ProcedureFuture.java:80)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
	at mkdb.Worker.run(Worker.java:32)

2016-09-09 14:47:20:903 ERROR BATTLE:mkdb.Worker.procedure.51 - 在角色4097下线的情况下，使用了该角色的ERole。因为ERole是内存表，数据可能没有  ，请根据堆栈内容确认此使用是否正确。
2016-09-09 14:47:20:903 ERROR BATTLE:mkdb.Worker.procedure.51 - 	at java.lang.Thread.getStackTrace(Unknown Source)
	at fire.pb.effect.RoleImpl.<init>(RoleImpl.java:80)
	at fire.pb.buff.BuffRoleImpl.getERole(BuffRoleImpl.java:252)
	at fire.pb.buff.continual.ConstantlyBuff.impact(ConstantlyBuff.java:102)
	at fire.pb.buff.BuffAgent.addCBuff(BuffAgent.java:294)
	at fire.pb.skill.SceneSkillRole.equip(SceneSkillRole.java:220)
	at fire.pb.role.PCreateRole.process(PCreateRole.java:503)
	at mkdb.Procedure.call(Procedure.java:109)
	at mkdb.Transaction.perform(Transaction.java:279)
	at mkdb.ProcedureFuture.run(ProcedureFuture.java:80)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
	at mkdb.Worker.run(Worker.java:32)

2016-09-09 14:47:21:013 ERROR BATTLE:mkdb.Worker.procedure.51 - 在角色4097下线的情况下，使用了该角色的ERole。因为ERole是内存表，数据可能没有  ，请根据堆栈内容确认此使用是否正确。
2016-09-09 14:47:21:013 ERROR BATTLE:mkdb.Worker.procedure.51 - 	at java.lang.Thread.getStackTrace(Unknown Source)
	at fire.pb.effect.RoleImpl.<init>(RoleImpl.java:80)
	at fire.pb.skill.SceneSkillRole.equip(SceneSkillRole.java:165)
	at fire.pb.role.PCreateRole.process(PCreateRole.java:503)
	at mkdb.Procedure.call(Procedure.java:109)
	at mkdb.Transaction.perform(Transaction.java:279)
	at mkdb.ProcedureFuture.run(ProcedureFuture.java:80)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
	at mkdb.Worker.run(Worker.java:32)

2016-09-09 14:47:21:013 ERROR BATTLE:mkdb.Worker.procedure.51 - 在角色4097下线的情况下，使用了该角色的ERole。因为ERole是内存表，数据可能没有  ，请根据堆栈内容确认此使用是否正确。
2016-09-09 14:47:21:013 ERROR BATTLE:mkdb.Worker.procedure.51 - 	at java.lang.Thread.getStackTrace(Unknown Source)
	at fire.pb.effect.RoleImpl.<init>(RoleImpl.java:80)
	at fire.pb.buff.BuffRoleImpl.getERole(BuffRoleImpl.java:252)
	at fire.pb.buff.continual.ConstantlyBuff.impact(ConstantlyBuff.java:102)
	at fire.pb.buff.BuffAgent.addCBuff(BuffAgent.java:294)
	at fire.pb.skill.SceneSkillRole.equip(SceneSkillRole.java:220)
	at fire.pb.role.PCreateRole.process(PCreateRole.java:503)
	at mkdb.Procedure.call(Procedure.java:109)
	at mkdb.Transaction.perform(Transaction.java:279)
	at mkdb.ProcedureFuture.run(ProcedureFuture.java:80)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
	at mkdb.Worker.run(Worker.java:32)

2016-09-09 14:47:21:013 ERROR BATTLE:mkdb.Worker.procedure.51 - 在角色4097下线的情况下，使用了该角色的ERole。因为ERole是内存表，数据可能没有  ，请根据堆栈内容确认此使用是否正确。
2016-09-09 14:47:21:013 ERROR BATTLE:mkdb.Worker.procedure.51 - 	at java.lang.Thread.getStackTrace(Unknown Source)
	at fire.pb.effect.RoleImpl.<init>(RoleImpl.java:80)
	at fire.pb.skill.SceneSkillRole.equip(SceneSkillRole.java:165)
	at fire.pb.role.PCreateRole.process(PCreateRole.java:503)
	at mkdb.Procedure.call(Procedure.java:109)
	at mkdb.Transaction.perform(Transaction.java:279)
	at mkdb.ProcedureFuture.run(ProcedureFuture.java:80)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
	at mkdb.Worker.run(Worker.java:32)

2016-09-09 14:47:21:013 ERROR BATTLE:mkdb.Worker.procedure.51 - 在角色4097下线的情况下，使用了该角色的ERole。因为ERole是内存表，数据可能没有  ，请根据堆栈内容确认此使用是否正确。
2016-09-09 14:47:21:013 ERROR BATTLE:mkdb.Worker.procedure.51 - 	at java.lang.Thread.getStackTrace(Unknown Source)
	at fire.pb.effect.RoleImpl.<init>(RoleImpl.java:80)
	at fire.pb.buff.BuffRoleImpl.getERole(BuffRoleImpl.java:252)
	at fire.pb.buff.continual.ConstantlyBuff.impact(ConstantlyBuff.java:102)
	at fire.pb.buff.BuffAgent.addCBuff(BuffAgent.java:294)
	at fire.pb.skill.SceneSkillRole.equip(SceneSkillRole.java:220)
	at fire.pb.role.PCreateRole.process(PCreateRole.java:503)
	at mkdb.Procedure.call(Procedure.java:109)
	at mkdb.Transaction.perform(Transaction.java:279)
	at mkdb.ProcedureFuture.run(ProcedureFuture.java:80)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
	at mkdb.Worker.run(Worker.java:32)

2016-09-09 14:47:21:013 ERROR BATTLE:mkdb.Worker.procedure.51 - 在角色4097下线的情况下，使用了该角色的ERole。因为ERole是内存表，数据可能没有  ，请根据堆栈内容确认此使用是否正确。
2016-09-09 14:47:21:013 ERROR BATTLE:mkdb.Worker.procedure.51 - 	at java.lang.Thread.getStackTrace(Unknown Source)
	at fire.pb.effect.RoleImpl.<init>(RoleImpl.java:80)
	at fire.pb.skill.SceneSkillRole.equip(SceneSkillRole.java:165)
	at fire.pb.role.PCreateRole.process(PCreateRole.java:503)
	at mkdb.Procedure.call(Procedure.java:109)
	at mkdb.Transaction.perform(Transaction.java:279)
	at mkdb.ProcedureFuture.run(ProcedureFuture.java:80)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
	at mkdb.Worker.run(Worker.java:32)

2016-09-09 14:47:21:013 ERROR BATTLE:mkdb.Worker.procedure.51 - 在角色4097下线的情况下，使用了该角色的ERole。因为ERole是内存表，数据可能没有  ，请根据堆栈内容确认此使用是否正确。
2016-09-09 14:47:21:013 ERROR BATTLE:mkdb.Worker.procedure.51 - 	at java.lang.Thread.getStackTrace(Unknown Source)
	at fire.pb.effect.RoleImpl.<init>(RoleImpl.java:80)
	at fire.pb.buff.BuffRoleImpl.getERole(BuffRoleImpl.java:252)
	at fire.pb.buff.continual.ConstantlyBuff.impact(ConstantlyBuff.java:102)
	at fire.pb.buff.BuffAgent.addCBuff(BuffAgent.java:294)
	at fire.pb.skill.SceneSkillRole.equip(SceneSkillRole.java:220)
	at fire.pb.role.PCreateRole.process(PCreateRole.java:503)
	at mkdb.Procedure.call(Procedure.java:109)
	at mkdb.Transaction.perform(Transaction.java:279)
	at mkdb.ProcedureFuture.run(ProcedureFuture.java:80)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
	at mkdb.Worker.run(Worker.java:32)

2016-09-09 14:47:21:013 ERROR BATTLE:mkdb.Worker.procedure.51 - 在角色4097下线的情况下，使用了该角色的ERole。因为ERole是内存表，数据可能没有  ，请根据堆栈内容确认此使用是否正确。
2016-09-09 14:47:21:013 ERROR BATTLE:mkdb.Worker.procedure.51 - 	at java.lang.Thread.getStackTrace(Unknown Source)
	at fire.pb.effect.RoleImpl.<init>(RoleImpl.java:80)
	at fire.pb.skill.SceneSkillRole.equip(SceneSkillRole.java:165)
	at fire.pb.role.PCreateRole.process(PCreateRole.java:503)
	at mkdb.Procedure.call(Procedure.java:109)
	at mkdb.Transaction.perform(Transaction.java:279)
	at mkdb.ProcedureFuture.run(ProcedureFuture.java:80)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
	at mkdb.Worker.run(Worker.java:32)

2016-09-09 14:47:21:013 ERROR BATTLE:mkdb.Worker.procedure.51 - 在角色4097下线的情况下，使用了该角色的ERole。因为ERole是内存表，数据可能没有  ，请根据堆栈内容确认此使用是否正确。
2016-09-09 14:47:21:013 ERROR BATTLE:mkdb.Worker.procedure.51 - 	at java.lang.Thread.getStackTrace(Unknown Source)
	at fire.pb.effect.RoleImpl.<init>(RoleImpl.java:80)
	at fire.pb.buff.BuffRoleImpl.getERole(BuffRoleImpl.java:252)
	at fire.pb.buff.continual.ConstantlyBuff.impact(ConstantlyBuff.java:102)
	at fire.pb.buff.BuffAgent.addCBuff(BuffAgent.java:294)
	at fire.pb.skill.SceneSkillRole.equip(SceneSkillRole.java:220)
	at fire.pb.role.PCreateRole.process(PCreateRole.java:503)
	at mkdb.Procedure.call(Procedure.java:109)
	at mkdb.Transaction.perform(Transaction.java:279)
	at mkdb.ProcedureFuture.run(ProcedureFuture.java:80)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
	at mkdb.Worker.run(Worker.java:32)

2016-09-09 14:47:21:060 ERROR BATTLE:mkdb.Worker.procedure.51 - 在角色4097下线的情况下，使用了该角色的ERole。因为ERole是内存表，数据可能没有  ，请根据堆栈内容确认此使用是否正确。
2016-09-09 14:47:21:060 ERROR BATTLE:mkdb.Worker.procedure.51 - 	at java.lang.Thread.getStackTrace(Unknown Source)
	at fire.pb.effect.RoleImpl.<init>(RoleImpl.java:80)
	at fire.pb.effect.RoleImpl.<init>(RoleImpl.java:71)
	at fire.pb.PInitRoleProp.process(PInitRoleProp.java:16)
	at mkdb.Procedure.call(Procedure.java:109)
	at fire.pb.role.PCreateRole.process(PCreateRole.java:509)
	at mkdb.Procedure.call(Procedure.java:109)
	at mkdb.Transaction.perform(Transaction.java:279)
	at mkdb.ProcedureFuture.run(ProcedureFuture.java:80)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
	at mkdb.Worker.run(Worker.java:32)

2016-09-09 14:47:23:960 ERROR BATTLE:mkdb.Worker.procedure.51 - 在角色4097下线的情况下，使用了该角色的ERole。因为ERole是内存表，数据可能没有  ，请根据堆栈内容确认此使用是否正确。
2016-09-09 14:47:23:976 ERROR BATTLE:mkdb.Worker.procedure.51 - 	at java.lang.Thread.getStackTrace(Unknown Source)
	at fire.pb.effect.RoleImpl.<init>(RoleImpl.java:80)
	at fire.pb.buff.BuffRoleImpl.getERole(BuffRoleImpl.java:252)
	at fire.pb.buff.continual.ConstantlyBuff.impact(ConstantlyBuff.java:102)
	at fire.pb.buff.BuffAgent.addCBuff(BuffAgent.java:294)
	at fire.pb.skill.SkillRole.addSkillBaseBuff(SkillRole.java:941)
	at fire.pb.skill.SkillRole.updateInbornWithSP(SkillRole.java:455)
	at fire.pb.role.PCreateRole.process(PCreateRole.java:533)
	at mkdb.Procedure.call(Procedure.java:109)
	at mkdb.Transaction.perform(Transaction.java:279)
	at mkdb.ProcedureFuture.run(ProcedureFuture.java:80)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
	at mkdb.Worker.run(Worker.java:32)

2016-09-09 14:47:24:054 ERROR BATTLE:mkdb.Worker.procedure.51 - 在角色4097下线的情况下，使用了该角色的ERole。因为ERole是内存表，数据可能没有  ，请根据堆栈内容确认此使用是否正确。
2016-09-09 14:47:24:054 ERROR BATTLE:mkdb.Worker.procedure.51 - 	at java.lang.Thread.getStackTrace(Unknown Source)
	at fire.pb.effect.RoleImpl.<init>(RoleImpl.java:80)
	at fire.pb.buff.BuffRoleImpl.getERole(BuffRoleImpl.java:252)
	at fire.pb.buff.continual.ConstantlyBuff.closure(ConstantlyBuff.java:153)
	at fire.pb.buff.BuffAgent.removeCBuff(BuffAgent.java:394)
	at fire.pb.buff.BuffAgent.processOverrideSelf(BuffAgent.java:305)
	at fire.pb.buff.BuffAgent.addCBuff(BuffAgent.java:283)
	at fire.pb.skill.SkillRole.addSkillBaseBuff(SkillRole.java:941)
	at fire.pb.skill.SkillRole.updateInbornWithSP(SkillRole.java:455)
	at fire.pb.role.PCreateRole.process(PCreateRole.java:533)
	at mkdb.Procedure.call(Procedure.java:109)
	at mkdb.Transaction.perform(Transaction.java:279)
	at mkdb.ProcedureFuture.run(ProcedureFuture.java:80)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
	at mkdb.Worker.run(Worker.java:32)

2016-09-09 14:47:24:054 ERROR BATTLE:mkdb.Worker.procedure.51 - 在角色4097下线的情况下，使用了该角色的ERole。因为ERole是内存表，数据可能没有  ，请根据堆栈内容确认此使用是否正确。
2016-09-09 14:47:24:054 ERROR BATTLE:mkdb.Worker.procedure.51 - 	at java.lang.Thread.getStackTrace(Unknown Source)
	at fire.pb.effect.RoleImpl.<init>(RoleImpl.java:80)
	at fire.pb.buff.BuffRoleImpl.getERole(BuffRoleImpl.java:252)
	at fire.pb.buff.continual.ConstantlyBuff.impact(ConstantlyBuff.java:102)
	at fire.pb.buff.BuffAgent.addCBuff(BuffAgent.java:294)
	at fire.pb.skill.SkillRole.addSkillBaseBuff(SkillRole.java:941)
	at fire.pb.skill.SkillRole.updateInbornWithSP(SkillRole.java:455)
	at fire.pb.role.PCreateRole.process(PCreateRole.java:533)
	at mkdb.Procedure.call(Procedure.java:109)
	at mkdb.Transaction.perform(Transaction.java:279)
	at mkdb.ProcedureFuture.run(ProcedureFuture.java:80)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
	at mkdb.Worker.run(Worker.java:32)

2016-09-09 14:47:24:054 ERROR BATTLE:mkdb.Worker.procedure.51 - 在角色4097下线的情况下，使用了该角色的ERole。因为ERole是内存表，数据可能没有  ，请根据堆栈内容确认此使用是否正确。
2016-09-09 14:47:24:054 ERROR BATTLE:mkdb.Worker.procedure.51 - 	at java.lang.Thread.getStackTrace(Unknown Source)
	at fire.pb.effect.RoleImpl.<init>(RoleImpl.java:80)
	at fire.pb.buff.BuffRoleImpl.getERole(BuffRoleImpl.java:252)
	at fire.pb.buff.continual.ConstantlyBuff.closure(ConstantlyBuff.java:153)
	at fire.pb.buff.BuffAgent.removeCBuff(BuffAgent.java:394)
	at fire.pb.buff.BuffAgent.processOverrideSelf(BuffAgent.java:305)
	at fire.pb.buff.BuffAgent.addCBuff(BuffAgent.java:283)
	at fire.pb.skill.SkillRole.addSkillBaseBuff(SkillRole.java:941)
	at fire.pb.skill.SkillRole.updateInbornWithSP(SkillRole.java:455)
	at fire.pb.role.PCreateRole.process(PCreateRole.java:533)
	at mkdb.Procedure.call(Procedure.java:109)
	at mkdb.Transaction.perform(Transaction.java:279)
	at mkdb.ProcedureFuture.run(ProcedureFuture.java:80)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
	at mkdb.Worker.run(Worker.java:32)

2016-09-09 14:47:24:054 ERROR BATTLE:mkdb.Worker.procedure.51 - 在角色4097下线的情况下，使用了该角色的ERole。因为ERole是内存表，数据可能没有  ，请根据堆栈内容确认此使用是否正确。
2016-09-09 14:47:24:054 ERROR BATTLE:mkdb.Worker.procedure.51 - 	at java.lang.Thread.getStackTrace(Unknown Source)
	at fire.pb.effect.RoleImpl.<init>(RoleImpl.java:80)
	at fire.pb.buff.BuffRoleImpl.getERole(BuffRoleImpl.java:252)
	at fire.pb.buff.continual.ConstantlyBuff.impact(ConstantlyBuff.java:102)
	at fire.pb.buff.BuffAgent.addCBuff(BuffAgent.java:294)
	at fire.pb.skill.SkillRole.addSkillBaseBuff(SkillRole.java:941)
	at fire.pb.skill.SkillRole.updateInbornWithSP(SkillRole.java:455)
	at fire.pb.role.PCreateRole.process(PCreateRole.java:533)
	at mkdb.Procedure.call(Procedure.java:109)
	at mkdb.Transaction.perform(Transaction.java:279)
	at mkdb.ProcedureFuture.run(ProcedureFuture.java:80)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
	at mkdb.Worker.run(Worker.java:32)

2016-09-09 14:47:24:632 ERROR BATTLE:mkdb.Worker.procedure.34 - 在角色4097下线的情况下，使用了该角色的ERole。因为ERole是内存表，数据可能没有  ，请根据堆栈内容确认此使用是否正确。
2016-09-09 14:47:24:632 ERROR BATTLE:mkdb.Worker.procedure.34 - 	at java.lang.Thread.getStackTrace(Unknown Source)
	at fire.pb.effect.RoleImpl.<init>(RoleImpl.java:80)
	at fire.pb.buff.BuffRoleImpl.getERole(BuffRoleImpl.java:252)
	at fire.pb.buff.continual.ConstantlyBuff.impact(ConstantlyBuff.java:102)
	at fire.pb.buff.BuffAgent.addCBuff(BuffAgent.java:294)
	at fire.pb.buff.BuffAgent.addCBuffWithSP(BuffAgent.java:359)
	at fire.pb.role.PAddBuffToCreatRoleProc.process(PAddBuffToCreatRoleProc.java:24)
	at mkdb.Procedure.call(Procedure.java:109)
	at mkdb.Transaction.perform(Transaction.java:279)
	at mkdb.ProcedureFuture.run(ProcedureFuture.java:80)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
	at mkdb.Worker.run(Worker.java:32)

2016-09-09 14:47:24:632 ERROR BATTLE:mkdb.Worker.procedure.34 - 在角色4097下线的情况下，使用了该角色的ERole。因为ERole是内存表，数据可能没有  ，请根据堆栈内容确认此使用是否正确。
2016-09-09 14:47:24:632 ERROR BATTLE:mkdb.Worker.procedure.34 - 	at java.lang.Thread.getStackTrace(Unknown Source)
	at fire.pb.effect.RoleImpl.<init>(RoleImpl.java:80)
	at fire.pb.buff.BuffRoleImpl.getERole(BuffRoleImpl.java:252)
	at fire.pb.buff.continual.ConstantlyBuff.impact(ConstantlyBuff.java:102)
	at fire.pb.buff.BuffAgent.addCBuff(BuffAgent.java:294)
	at fire.pb.buff.BuffAgent.addCBuffWithSP(BuffAgent.java:359)
	at fire.pb.role.PAddBuffToCreatRoleProc.process(PAddBuffToCreatRoleProc.java:29)
	at mkdb.Procedure.call(Procedure.java:109)
	at mkdb.Transaction.perform(Transaction.java:279)
	at mkdb.ProcedureFuture.run(ProcedureFuture.java:80)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
	at mkdb.Worker.run(Worker.java:32)

2016-09-09 14:56:23:382  INFO BATTLE:main - effect模块初始化开始。
2016-09-09 14:56:24:241  INFO BATTLE:main - effect模块初始化完成。
2016-09-09 14:56:24:285  INFO BATTLE:main - battle模块初始化开始
2016-09-09 14:56:24:379  INFO BATTLE:main - battle阵型信息初始化完成
2016-09-09 14:56:24:997  INFO BATTLE:main - battle AI信息初始化完成
2016-09-09 14:56:25:068  INFO BATTLE:main - 战场信息初始化完成
2016-09-09 14:56:25:068  INFO BATTLE:main - battle模块初始化完成
2016-09-09 14:56:27:466  INFO BATTLE:main - BUFF模块初始化开始
2016-09-09 14:56:27:998  INFO BATTLE:main - BUFF模块初始化完成
2016-09-09 14:59:48:431  INFO BATTLE:main - effect模块初始化开始。
2016-09-09 14:59:49:298  INFO BATTLE:main - effect模块初始化完成。
2016-09-09 14:59:49:345  INFO BATTLE:main - battle模块初始化开始
2016-09-09 14:59:49:392  INFO BATTLE:main - battle阵型信息初始化完成
2016-09-09 14:59:49:782  INFO BATTLE:main - battle AI信息初始化完成
2016-09-09 14:59:49:804  INFO BATTLE:main - 战场信息初始化完成
2016-09-09 14:59:49:804  INFO BATTLE:main - battle模块初始化完成
2016-09-09 14:59:52:189  INFO BATTLE:main - BUFF模块初始化开始
2016-09-09 14:59:52:689  INFO BATTLE:main - BUFF模块初始化完成
